internal/workload/podman/podman.go

```diff
@@ -77,3 +79,32 @@ func (p *Podman) Start(workloadId string) error {
 	}
 	return nil
 }
+
+func (p *Podman) ListSecrets() (map[string]interface{}, error) {
```

Comment:
Why is the value `interface{}` here?

Reply:
OK. `map[string]struct{}` would be more memory-optimal and not so confusing - currently the values in the ma are always `nil` (https://github.com/jakub-dzon/k4e-device-worker/pull/55/files#diff-a98fd4c429032f68a1a6597b03c3ccba5f014918e82f41108aa962180f36e13eR90).

Reply:
changed